include/exadg/grid/grid_motion_function.h

```diff
@@ -48,16 +48,19 @@ class GridMotionFunction : public GridMotionBase<dim, Number>
       mesh_movement_function(mesh_movement_function),
       triangulation(triangulation)
   {
-    update(start_time, false);
+    update(start_time, false, dealii::numbers::invalid_unsigned_int);
```

Comment:
you prefer 0 whenever I put `dealii::numbers::invalid_unsigned int`? it is `static_cast<unsigned int>(-1)`, this way one would now immediately get that the `time_step_number` is fishy when debugging (and having an enormous unsigned int unexpectedly)!